src/functionalTest/resources/scenarios/RIA-3302-notification-for-end-appeal-on-state-appeal-submitted-co.json

```diff
@@ -0,0 +1,34 @@
+{
+  "description": "RIA 3302 sends notification for end appeal on state appeal submitted by case officer",
```

Comment:
Could you add a functional test covering the event for 'Judge' user? Also could you use different endAppealOutcome in the test data to give it more coverage instead of just using 'Struck out'?

Reply:
Added test for Judge and different endAppealOutcome

Reply:
Thanks Lee.